core/pkg/resourcehandler/remotegitutils.go

```diff
@@ -40,7 +40,7 @@ func isGitTokenPresent(gitURL giturl.IGitAPI) bool {
 
 // Get the error message according to the provider
 func getProviderError(gitURL giturl.IGitAPI) error {
-	switch gitURL.GetProvider(){
+	switch gitURL.GetProvider() {
```

Comment:
are these needed to fix linting? can we have a separate commit for all formatting related changes?

Reply:
Yes, they are. And we do have a separate commit for that inside this pull request :)